pxr/imaging/plugin/hdRpr/distantLight.cpp

```diff
@@ -61,7 +61,6 @@ void HdRprDistantLight::Sync(HdSceneDelegate* sceneDelegate,
                 *dirtyBits = HdLight::Clean;
                 return;
             }
-            rprRenderParam->AddLight();
```

Comment:
I thought this was for dome light?  Or is it removing some default distant light?

Reply:
That was about [tracking the number of lights](https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD/blob/master/pxr/imaging/plugin/hdRpr/renderParam.h#L53-L55) on the scene so that we can decide whether we need default light or not. It has been moved to HdRprApi class